app/helpers/icons_helper.rb

```diff
@@ -19,25 +19,37 @@ def language_icon(language)
 
   private
 
-  def status_fa_icon(status)
-    fa_icon(*icon_for_status(status))
+  def contextualization_fa_icon(contextualization)
+    fa_icon(*icon_for(contextualization))
   end
 
   def exercise_status_fa_icon(exercise)
-    status_fa_icon(exercise.status_for(current_user))
+    contextualization_fa_icon(exercise.assignment_for(current_user))
   end
 
   def discussion_status_fa_icon(discussion)
-    status_fa_icon(discussion.status)
+    contextualization_fa_icon(discussion)
   end
 
-  def icon_for_status(s)
-    iconized = s.iconize
+  def icon_for(iconizable)
```

Comment:
Iconizable is not a word so we should change it. I hear suggestions :smile:.

Reply:
I think iconizable is the clearest choice.